bundles/org.openhab.binding.daikin/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -18,6 +18,33 @@
 			<channel id="homekitmode" typeId="acunit-homekitmode"/>
 			<channel id="fanspeed" typeId="acunit-fan"/>
 			<channel id="fandir" typeId="acunit-fandir"/>
+			<channel id="cmpfrequency" typeId="acunit-cmpfrequency"></channel>
+			<channel id="specialmode" typeId="acunit-specialmode"></channel>
+			<channel id="specialmode-powerful" typeId="acunit-specialmode-powerful"></channel>
+			<channel id="energyheatingcurrentyear-1" typeId="acunit-energyheatingcurrentyear-1"></channel>
+			<channel id="energyheatingcurrentyear-2" typeId="acunit-energyheatingcurrentyear-2"></channel>
+			<channel id="energyheatingcurrentyear-3" typeId="acunit-energyheatingcurrentyear-3"></channel>
+			<channel id="energyheatingcurrentyear-4" typeId="acunit-energyheatingcurrentyear-4"></channel>
+			<channel id="energyheatingcurrentyear-5" typeId="acunit-energyheatingcurrentyear-5"></channel>
+			<channel id="energyheatingcurrentyear-6" typeId="acunit-energyheatingcurrentyear-6"></channel>
+			<channel id="energyheatingcurrentyear-7" typeId="acunit-energyheatingcurrentyear-7"></channel>
+			<channel id="energyheatingcurrentyear-8" typeId="acunit-energyheatingcurrentyear-8"></channel>
+			<channel id="energyheatingcurrentyear-9" typeId="acunit-energyheatingcurrentyear-9"></channel>
+			<channel id="energyheatingcurrentyear-10" typeId="acunit-energyheatingcurrentyear-10"></channel>
+			<channel id="energyheatingcurrentyear-11" typeId="acunit-energyheatingcurrentyear-11"></channel>
+			<channel id="energyheatingcurrentyear-12" typeId="acunit-energyheatingcurrentyear-12"></channel>
+			<channel id="energycoolingcurrentyear-1" typeId="acunit-energycoolingcurrentyear-1"></channel>
+			<channel id="energycoolingcurrentyear-2" typeId="acunit-energycoolingcurrentyear-2"></channel>
+			<channel id="energycoolingcurrentyear-3" typeId="acunit-energycoolingcurrentyear-3"></channel>
+			<channel id="energycoolingcurrentyear-4" typeId="acunit-energycoolingcurrentyear-4"></channel>
+			<channel id="energycoolingcurrentyear-5" typeId="acunit-energycoolingcurrentyear-5"></channel>
+			<channel id="energycoolingcurrentyear-6" typeId="acunit-energycoolingcurrentyear-6"></channel>
+			<channel id="energycoolingcurrentyear-7" typeId="acunit-energycoolingcurrentyear-7"></channel>
+			<channel id="energycoolingcurrentyear-8" typeId="acunit-energycoolingcurrentyear-8"></channel>
+			<channel id="energycoolingcurrentyear-9" typeId="acunit-energycoolingcurrentyear-9"></channel>
+			<channel id="energycoolingcurrentyear-10" typeId="acunit-energycoolingcurrentyear-10"></channel>
+			<channel id="energycoolingcurrentyear-11" typeId="acunit-energycoolingcurrentyear-11"></channel>
+			<channel id="energycoolingcurrentyear-12" typeId="acunit-energycoolingcurrentyear-12"></channel>
```

Comment:
Perhaps you should make two channel groups; one for `energyheating` and another for `energycooling`.

Reply:
From the openhab documentation my understanding is either direct channels or channel groups. Am I wrong?
[openhab docs](https://www.openhab.org/docs/developer/bindings/thing-xml.html#channel-groups)

> A thing can only have direct channels or channel groups, but not both

Reply:
Ok, I was not aware of that restriction. It doesn't make much sense to me why it would be like that so I'll have to look into it. Thanks for pointing that out.